src/Chapter2.hs

```diff
@@ -740,7 +767,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n arr = filter (\x -> elem n x) arr
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to other functions as well  🙂 
But your implementation is already great 💯 

Here you have two opportunities for eta-reduction: the last argument to `filter` and the lambda function